lib/isFunction.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Is Function
```

Comment:
Can you remove these headers in the docs (here and elsewhere)? They don't seem that useful to me. It's just a repeat of the name of the function. It doesn't really fit with the documentation style of the rest of the code base.

Reply:
That’s a fair point, most of the time I prefer hindley-milner type like:

```js
/**
 * isFunction :: Function -> Boolean
 */
```

but that felt unwelcomed in this project, especially overlapping with JSDOC so I fell back on my previous habit of a human-readable name. I have no defence of this, will delete.